docs/book/src/SUMMARY.md

```diff
@@ -73,6 +73,10 @@
   - [Behavior Considered Undefined](./reference/undefined_behavior.md)
   - [Differences From Solidity](./reference/solidity_differences.md)
   - [Differences From Rust](./reference/rust_differences.md)
+  - [Storage](./reference/storage/storage.md)
+    - [Initialization](./reference/storage/init.md)
+    - [Reading & Writing](./reference/storage/read-write.md)
+    - [Libraries](./reference/storage/libraries/index.md)
```

Comment:
I think this error means you can't have these links nested here:

`AssertionError [ERR_ASSERTION]: cannot nest subfolders in SUMMARY.md`

```suggestion
```

Reply:
Will update the failing checks